src/components/content/catalog/services/details/ServiceDetail.tsx

```diff
@@ -83,6 +142,11 @@ function ServiceDetail({ serviceDetails }: { serviceDetails: ServiceTemplateDeta
                 <Descriptions.Item label='EULA'>
                     {serviceDetails.eula ? <AgreementText eula={serviceDetails.eula} /> : <span>Not Provided</span>}
                 </Descriptions.Item>
+                <Descriptions.Item label='Deployed Quantity'>
+                    <Button type='primary' shape='circle' danger onClick={onClick}>
```

Comment:
danger does not look really correct. we need something else
1. Any color other than red. 
2. It must be clear that the user can click and check the services -> probably something like this ` 2 <view services>` -> number of services is 2 and view services can be a text with link which will open the reports page.